CTFd/themes/core/templates/teams/public.html

```diff
@@ -75,14 +75,16 @@ <h3>Members</h3>
 					</thead>
 					<tbody>
 					{% for member in team.members %}
-						<tr>
-							<td>
-								<a href="{{ url_for('users.public', user_id=member.id) }}">
-									{{ member.name }}
-								</a>
-							</td>
-							<td>{{ member.score }}</td>
-						</tr>
+						{% if member.hidden != 1 and member.banned != 1 %}
```

Comment:
This logic looks very weird. 

The issue talks about:
> There probably needs to be a model property wrapping a function like awards and solves.

Hiding hidden members only in one area is likely not the right solution.